packages/camera/camera_windows/lib/camera_windows.dart

```diff
@@ -0,0 +1,437 @@
+    /// Windows camera plugin does not support resolution changed events.
+    return const Stream<CameraResolutionChangedEvent>.empty();
+  }
+
+  @override
+  Stream<CameraClosingEvent> onCameraClosing(int cameraId) {
+    return _cameraEvents(cameraId).whereType<CameraClosingEvent>();
+  }
+
+  @override
+  Stream<CameraErrorEvent> onCameraError(int cameraId) {
+    return _cameraEvents(cameraId).whereType<CameraErrorEvent>();
+  }
+
+  @override
+  Stream<VideoRecordedEvent> onVideoRecordedEvent(int cameraId) {
+    return _cameraEvents(cameraId).whereType<VideoRecordedEvent>();
+  }
+
+  @override
+  Stream<DeviceOrientationChangedEvent> onDeviceOrientationChanged() {
+    /// Windows camera plugin does not support capture orientations.
+    /// Force device orientation to landscape as by default camera plugin uses portraitUp orientation.
+    return Stream<DeviceOrientationChangedEvent>.value(
+      const DeviceOrientationChangedEvent(DeviceOrientation.landscapeRight),
+    );
+  }
+
+  @override
+  Future<void> lockCaptureOrientation(
+    int cameraId,
+    DeviceOrientation orientation,
+  ) async {
+    throw UnimplementedError('lockCaptureOrientation() is not implemented.');
+  }
+
+  @override
+  Future<void> unlockCaptureOrientation(int cameraId) async {
+    throw UnimplementedError('unlockCaptureOrientation() is not implemented.');
+  }
+
+  @override
+  Future<XFile> takePicture(int cameraId) async {
+    final String? path;
+    try {
+      path = await _pluginChannel.invokeMethod<String>(
+        'takePicture',
+        <String, dynamic>{'cameraId': cameraId},
+      );
+    } on PlatformException catch (e) {
+      throw CameraException(e.code, e.message);
+    }
+
+    return XFile(path!);
+  }
+
+  @override
+  Future<void> prepareForVideoRecording() async {
+    try {
+      _pluginChannel.invokeMethod<void>('prepareForVideoRecording');
+    } on PlatformException catch (e) {
+      throw CameraException(e.code, e.message);
+    }
+  }
+
+  @override
+  Future<void> startVideoRecording(
+    int cameraId, {
+    Duration? maxVideoDuration,
+  }) async {
+    try {
+      await _pluginChannel.invokeMethod<void>(
+        'startVideoRecording',
+        <String, dynamic>{
+          'cameraId': cameraId,
+          'maxVideoDuration': maxVideoDuration?.inMilliseconds,
+        },
+      );
+    } on PlatformException catch (e) {
+      throw CameraException(e.code, e.message);
+    }
+  }
+
+  @override
+  Future<XFile> stopVideoRecording(int cameraId) async {
+    final String? path;
+
+    try {
+      path = await _pluginChannel.invokeMethod<String>(
+        'stopVideoRecording',
+        <String, dynamic>{'cameraId': cameraId},
+      );
+    } on PlatformException catch (e) {
+      throw CameraException(e.code, e.message);
+    }
+
+    return XFile(path!);
+  }
+
+  @override
+  Future<void> pauseVideoRecording(int cameraId) async {
+    throw UnimplementedError('pauseVideoRecording() is not implemented.');
```

Comment:
Similar here and below. Add TODOs that point to a bug or use Unsupported if we intend never to support this.